src/components/FormSnippet/FormSnippet.js

```diff
@@ -6,36 +6,62 @@ class FormSnippet extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      lang: props.lang,
-      title: props.title,
-      snippet: props.snippet,
+      snippet: {
+        lang: props.lang,
+        title: props.title,
+        code: props.code,
+      },
```

Comment:
wrong props value, please see `FormSnippet.defaultProps`

Reply:
```js
this.state = {
  snippet: props.snippet,
};
```
What do you think ?

Reply:
My fault, sorry